subsys/fs/fuse_fs_access.c

```diff
@@ -176,9 +176,15 @@ static int fuse_fs_access_readdir(const char *path, void *buf,
 		 * directory but FUSE strips the trailing slashes from
 		 * directory names so add it back.
 		 */
-		char mount_path[PATH_MAX];
+		char mount_path[PATH_MAX] = {0};
+		size_t len = strlen(path);
 
-		sprintf(mount_path, "%s/", path);
+		if (len >= (PATH_MAX - 2)) {
+			return -ENOMEM;
+		}
+
+		memcpy(mount_path, path, len)
```

Comment:
missing semicolon, did this patch pass CI?

Reply:
clearly the CI is not running this code path, could you send a PR with the fix?